index.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (headerText, data) {
     var template = data === false ? headerText : gutil.template(headerText, extend({ file: file, filename: filename }, data));
     concat = new Concat(true, filename);
 
-    if (fs.lstatSync(file.path).isDirectory()) {
+    if (fs.existsSync(file.path) && fs.lstatSync(file.path).isDirectory()) {
```

Comment:
Frankly I'd rather have the library working than worry about this particular nitpiack. gulp-angular-templatecache is completely broken for us because of this bug and it's been preventing my employer from creating builds all day long.